test/parallel/test-benchmark-fs.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('fs', [
+  "n=1",
+  "size=1",
+  "dur=1",
+  "filesize=1024"
+]);
```

Comment:
Is the CI passing here? I would expect it to fail for e.g. `benchmark/fs/bench-realpath.js` as it requires a type that is not set here.

Reply:
The benchmark will run with the default values given to `common.createBenchmark` for any argument that is not specified. So in the case of `benchmark/fs/bench-realpath.js`, both the `relative` and `resolved` types are being run. I figured that since the purpose of pulling these benchmarks into the test suite was to ensure they don't break, it would be good to go through each of the code paths.

Reply:
Hello @charlieduong94,
What you say is true, and ideally that is what we would want, but that make this a very long test, and we are trying to strike a balance between coverage and manageability of the test suite.

Running this test on our fastest machine took 10 seconds (ignore the `_ms` in the log)
```
ok 242 parallel/test-benchmark-fs
  ---
  duration_ms: 10.107
  ...
```
I'm assuming that if we run this on only a single type it should take ~33% of that time. (Personally I'd pick `buf`)

Anyway this is a good PR, so thank you very much.

Reply:
I can make that change if you would like. Some minor changes will need to be made to the `read-stream-throughput` benchmark to get the `type` arg to play nicely with each other.